src/helpers/grapher.js

```diff
@@ -139,7 +139,7 @@ export function Graph(props) {
   testVariables.forEach((varSet, i) => {
     var options = [];
     let seperatedTestCase = currentTestCase.split("-");
-    console.log(seperatedTestCase);
+    //console.log(seperatedTestCase);
```

Comment:
Should just remove the console logs entirely. Not much reason to keep them here commented out.